tests/conftest.py

```diff
@@ -243,3 +255,74 @@ def _agent_profile(paw=None, group='red', platform='linux', executors=None, priv
         )
 
     return _agent_profile
+
+
+@pytest.fixture
+def api_client(loop, aiohttp_client):
```

Comment:
if this api client is only being used for API v2, maybe we can rename it to `api_v2_client` so it doesn't get mixed up with other api clients in other tests